### PR TITLE
update uuid

### PR DESCRIPTION
**Pull request by mina-skunk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/kelektiv/node-uuid#quickstart---commonjs-recommended
https://github.com/kelektiv/node-uuid/blob/master/v5.js#L47
https://github.com/kelektiv/node-uuid/blob/master/v5.js#L48
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by cjbarth** [comment]
Why not type set the v5Static values for URL and DNS to the GUID values instead of typing them as `string`?

**Comment by cjbarth** [comment]
I've mentioned an issue that needs attention, so I don't think it should be automatically merged.

File: types/uuid/uuid-tests.ts [hunk 1]
```diff
@@ -48,3 +48,7 @@ const a: string = v5('hello', MY_NAMESPACE);
 const b: string = v5('world', MY_NAMESPACE);
 const c: Buffer = v5('world', MY_NAMESPACE, new Buffer(16));
 const d: number[] = v5('world', MY_NAMESPACE, [], 0);
+
+// https://github.com/kelektiv/node-uuid#quickstart---commonjs-recommended
+const e = v5('hello.example.com', v5.DNS);
```
Comment (felipeochoa) [inline types/uuid/uuid-tests.ts]: I would add explicit types to `e` and `f` just as an extra test

**Comment by gatimus** [comment]
@cjbarth I though about that but wasn't sure if this would be an annoyance for developers, like so:

```ts
let myUuid = v5.DNS;
myUuid = v4(); // Type 'string' is not assignable to type '"6ba7b810-9dad-11d1-80b4-00c04fd430c8"'.
```

Also, isn't the point of the static values is that you don't have to know the standard, just that URL and DNS are UUID strings. If what the exact value of the URL and DNS UUIDs are, is within the scope of your code, should you be using a static from an outside module?

**Comment by felipeochoa** [comment]
Thanks for adding the type checks. I don't feel strongly about the string vs literal issue, but would be +0 on using `string`. Given how v5 uuids are computed, I don't think it would be a common use-case to manipulate the literal values directly.

**Review by felipeochoa — APPROVED** [review]

**Pull request merged**